src/OpenTelemetry.AutoInstrumentation.Native/startup_hook.h

```diff
@@ -12,20 +12,24 @@
 namespace trace
 {
 
+const WSTRING opentelemetry_autoinstrumentation_startuphook_filepath =
+    WStr("OpenTelemetry.AutoInstrumentation.StartupHook.dll");
+
 inline bool IsStartupHookValid(const std::vector<WSTRING>& startup_hooks, const WSTRING& home_path)
 {
     if (startup_hooks.empty())
     {
         return false;
     }
 
-    const auto expected_deployment_startuphook_path =
-        std::filesystem::path(home_path)
-        / "net" / "OpenTelemetry.AutoInstrumentation.StartupHook.dll";
-    const auto expected_nuget_startuphook_path = 
-        std::filesystem::path(home_path)
-        / "OpenTelemetry.AutoInstrumentation.StartupHook.dll";
-    if (!std::filesystem::exists(expected_deployment_startuphook_path) && !std::filesystem::exists(expected_nuget_startuphook_path))
+    auto startuphook_path = std::filesystem::path(home_path) / "net"
+                            / opentelemetry_autoinstrumentation_startuphook_filepath;
+    if (!std::filesystem::exists(startuphook_path))
+    {
+        startuphook_path = std::filesystem::path(home_path)
```

Comment:
can we add comment that it is case of nuget installation - previosly it was self-explained in `expected_nuget_startuphook_path` variable name.